examples/source/1.components/amp-img.html

```diff
@@ -81,15 +81,15 @@
   <!--
     `amp-img` supports AMP's [common attributes](/content/amp-dev/documentation/guides-and-tutorials/learn/common_attributes.md#fallback), which means you can show a `fallback` in case the image couldn't be loaded. This is great for adding offline support to your AMPs.
   -->
-  <amp-img src="/static/samples/img/does-not-exist.jpg" width="300" height="100" layout="responsive">
+  <amp-img src="/static/samples/img/does-not-exist.jpg" width="300" height="100" layout="responsive" alt="a non-existent image">
     <div fallback>offline</div>
   </amp-img>
 
   <!-- ## No script -->
   <!--
     If you want to ensure that users with JavaScript disabled can see your images, you can add an additional `img` element inside a `noscript` tag. This also makes it possible for webcrawlers without JavaScript runtime to discover your images.
   -->
-  <amp-img src="/static/samples/img/amp.jpg" alt="AMP" width="475" height="268" layout="responsive">
+  <amp-img src="/static/samples/img/amp.jpg" width="475" height="268" layout="responsive" alt="AMP">
```

Comment:
For my understanding, why was the `alt` moved to the end?

Reply:
Apologies, this was done purely for code/style consistency with all the other `<amp-img>` elements in this page. No accessibility reason per se.